charts/victoria-metrics-k8s-stack/templates/servicemonitors/kubedns.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if .Values.kubeDns.enabled }}
+{{- if .Values.kubeDns.service.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ printf "%s-%s" (include "victoria-metrics-k8s-stack.fullname" $) "kubedns" | trunc 63 | trimSuffix "-" }}
+  namespace: kube-system
+  labels:
+    app: {{ include "victoria-metrics-k8s-stack.fullname" . }}-kubedns
+    jobLabel: kubedns
+  {{ include "victoria-metrics-k8s-stack.labels" . | indent 4 }}
```

Comment:
this looks weird 2 spaces that indent 4, could we use only one approach

Reply:
fixed